server/db/dbcontrollers/stocks.js

```diff
@@ -0,0 +1,26 @@
+var methods = {};
```

Comment:
this file can be killed. was just used for testing before real stock data existed.